src/app/_components/entry-index-menu/index.tsx

```diff
@@ -83,19 +83,15 @@ export function EntryIndexMenu({ filter, tag }: Props) {
       href: `/rising/today`,
       selected: filter === "rising",
       id: "rising"
+    },
+    {
```

Comment:
We need to add promoted content into feed also, previously we injected promoted content every 3rd post in feeds.